lib/BridgeAbstract.php

```diff
@@ -8,13 +8,26 @@
 	const MAINTAINER = 'No maintainer';
 	const CACHE_TIMEOUT = 3600;
 	const PARAMETERS = array();
+	const FORMATS = array(); // array of compatibles formats or null
```

Comment:
I don't know is this is actually relevant : the bridges should be compatible with every formats, do you have an example case where that might not be true ?